config/routes.rb

```diff
@@ -1,7 +1,13 @@
 Rails.application.routes.draw do
+
   namespace :api do
     namespace :v0 do
       resources :pings, only: [:index], constraints: { format: 'json' }
     end
+
+    namespace :v1 do
+      resources :user, constraints: {format: 'json'}
```

Comment:
Resources are usually set in plural.

Reply:
Ok, this is good to know, When I tried to change it I got an `uninitialized constant API::V1:UserControlller` error. even after changing the name of the controller